test/test_other.py

```diff
@@ -1956,7 +1956,7 @@ def test_libpng(self):
     self.assertContained('libpng passes test', output)
 
   @node_pthreads
-  def test_libpng_with_pthreads(self):
+  def test_zzz_libpng_with_pthreads(self):
```

Comment:
Can we open a bug about this flakiness and link to it here?   That way we future readers will know why its got this funny name, and we can also rename it if/when the bug gets investigated and fixed.